src/e2e/java/teammates/e2e/pageobjects/AdminNotificationsPage.java

```diff
@@ -0,0 +1,232 @@
+
+        fillNotificationForm(notification);
+
+        clickEditNotificationButton();
+        waitForPageToLoad(true);
+    }
+
+    public void deleteNotification(NotificationAttributes notification) {
+        WebElement notificationRow = notificationsTable.findElement(By.id(notification.getNotificationId()));
+        WebElement deleteButton = notificationRow.findElement(By.className("btn-danger"));
+
+        deleteButton.click();
+        waitForConfirmationModalAndClickOk();
+        waitForPageToLoad(true);
+    }
+
+    public void fillNotificationForm(NotificationAttributes notification) {
+        selectDropdownOptionByText(notificationTargetUserDropdown, getTargetUserText(notification.getTargetUser()));
+        selectDropdownOptionByText(notificationStyleDropdown, getNotificationStyle(notification.getStyle()));
+        fillTextBox(notificationTitleTextBox, notification.getTitle());
+        setMessage(notification.getMessage());
+        setNotificationStartDateTime(notification.getStartTime());
+        setNotificationEndDateTime(notification.getEndTime());
+    }
+
+    public String getNewestNotificationId() {
+        WebElement latestNotificationRow = notificationsTable.findElement(By.className("table-success"));
+        return latestNotificationRow.getAttribute("id");
+    }
+
+    private void clickAddNotificationButton() {
+        click(addNotificationButton);
+    }
+
+    private void clickCreateNotificationButton() {
+        click(createNotificationButton);
+    }
+
+    private void clickEditNotificationButton() {
+        click(editNotificationButton);
+    }
+
+    private void setMessage(String message) {
+        writeToRichTextEditor(notificationMessageEditor, message);
+    }
+
+    private void setNotificationStartDateTime(Instant startInstant) {
+        setDateTime(startDateBox.findElement(By.tagName("input")), startTimeDropdown, startInstant);
+    }
+
+    private void setNotificationEndDateTime(Instant endInstant) {
+        setDateTime(endDateBox.findElement(By.tagName("input")), endTimeDropdown, endInstant);
+    }
+
+    private void setDateTime(WebElement dateBox, WebElement timeBox, Instant startInstant) {
+        fillTextBox(dateBox, getInputDateString(startInstant));
+        selectDropdownOptionByText(timeBox.findElement(By.tagName("select")), getInputTimeString(startInstant));
+    }
+
+    private String[] getNotificationTableDisplayDetails(NotificationAttributes notification) {
+        String[] details = new String[6];
+        details[0] = notification.getTitle();
+        details[1] = getTableDisplayDateString(notification.getStartTime());
+        details[2] = getTableDisplayDateString(notification.getEndTime());
+        details[3] = notification.getTargetUser().toString();
+        details[4] = getNotificationStyle(notification.getStyle());
+        details[5] = getTableDisplayDateString(notification.getCreatedAt());
+        return details;
```

Comment:
```suggestion
        return new String[] {
            notification.getTitle(), 
            getTableDisplayDateString(notification.getStartTime()),
            getTableDisplayDateString(notification.getEndTime()),
            notification.getTargetUser().toString(),
            getNotificationStyle(notification.getStyle()),
            getTableDisplayDateString(notification.getCreatedAt())
        };
```
Just a suggestion here: would it be better to declare the array instead of assigning value to each index?

Reply:
Thanks for the suggestion, have made the fix!